build_runner/lib/src/watcher/graph_watcher.dart

```diff
@@ -59,9 +59,9 @@ class PackageGraphWatcher {
                   'Error from directory watcher for package:${w.node.name}\n\n'
                   'If you see this consistently then it is recommended that '
                   'you enable the polling file watcher with '
-                  '--use-polling-watcher.');
-              // ignore: only_throw_errors
-              throw e;
+                  '--use-polling-watcher.',
+                  e,
```

Comment:
Is there a way to add a test for this?

Reply:
Not that I can think of - not anything reasonable at least :/